aliases/available/docker.aliases.bash

```diff
@@ -20,12 +19,17 @@ case $OSTYPE in
     ;;
 esac
 
-alias dkrmall='docker-remove-stale-assets'  # Delete all untagged images and exited containers
-alias dkrmli='docker-remove-most-recent-image'  # Delete most recent (i.e., last) Docker image
-alias dkrmi='docker-remove-images'  # Delete images for supplied IDs or all if no IDs are passed as arguments
-alias dkideps='docker-image-dependencies'  # Output a graph of image dependencies using Graphiz
-alias dkre='docker-runtime-environment'  # List environmental variables of the supplied image ID
-alias dkelc='docker exec -it `dklcid` bash' # Enter last container (works with Docker 1.3 and above)
+if [ ! -z "$(command ls "${BASH_IT}/$subdirectory/"{[0-9][0-9][0-9]${BASH_IT_LOAD_PRIORITY_SEPARATOR}docker,docker}.plugin.bash 2>/dev/null | head -1)" ]; then
```

Comment:
What is `$subdirectory` referring to here? I might have missed where this is set...

Reply:
Oops, seems my PR is missing a commit where I removed this. Was originally pasted from `lib/helpers.bash` in a slightly different context. Will correct as soon as I get home.